tinder.js

```diff
@@ -339,6 +365,28 @@ function TinderClient() {
       makeTinderCallback(callback));
   };
 
+
+  /**
+   * Post new user picture
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.uploadPicture = function(file, callback) {
+    tinderPost('image?',
+      null,
+      makeTinderCallback(callback),file);
+  };
+
+  /**
+   * Delete picture from user profile
+   * @param {String} pictureId the id of the picture
+   * @param {Function} callback the callback to invoke when the request completes
+   */
+  this.deletePicture = function(pictureId, callback) {
+    tinderDelete('media?',
+      {assets:[pictureId]},
```

Comment:
Can you expand this to full-syntax, as with the other JSON objects in the lib?

Also, curious, does it their API actually accept an array of pictureIds here? We should probably allow an array to be passed if so.